src/allmydata/client.py

```diff
@@ -211,8 +211,12 @@ def init_introducer_clients(self):
         # create a pool of introducer_clients
         self.introducer_clients = []
         for introducer_furl in self.introducer_furls:
-            tubID, location_hints, name = decode_furl(introducer_furl)
-            config_filepath = FilePath(os.path.join(self.basedir, "private", "%s.introduced.yaml" % name))
+            try:
+                tubID, location_hints, name = decode_furl(introducer_furl)
+                config_filepath = FilePath(os.path.join(self.basedir, "private", "%s.introduced.yaml" % name))
+            except ValueError, e:
+                config_filepath = FilePath(os.path.join(self.basedir, "private", "%s.introduced.yaml" % introducer_furl))
```

Comment:
if this is not the desired behavior and user interface for this feature-set then how shall we name the local yaml cache of announcements for each introducer?